cypress/e2e/functional/opal/paymentTerms/PO-471_collectionOrder_adultOrYouth.feature

```diff
@@ -0,0 +1,131 @@
+        Given I am on the OPAL Frontend
+        Then I see "Opal" in the header
+
+        When I sign in as "opal-test@HMCTS.NET"
+        And I am on the dashboard
+        And I navigate to Manual Account Creation
+
+        And I see "Business unit and defendant type" on the page header
+        And I enter "West London" into the business unit search box
+        And I select the "Fine" radio button
+        And I select the "Adult or youth only" radio button
+        And I click the "Continue" button
+        Then I see "Account details" on the page header
+        When I click on the "Personal details" link
+        And I see "Personal details" on the page header
+        And I select title "Mrs" from dropdown
+        And I enter "FNAME" into the "First names" field
+        And I enter "LNAME" into the "Last name" field
+        And I enter "ADDR1" into the "Address line 1" field
+
+    Scenario Outline: AC-1a, AC-2 positive: the 'Has a collection order been made?' field should be presented only if the defendant is an Adult
+        And I enter "<dateOfBirth>" into the Date of birth field
+        And I click the "Return to account details" button
+        Then I see the status of "Personal details" is "Provided"
+        And I click on the "Payment terms" link
+        Then I see "Payment terms" on the page header
+
+        And I select the "Yes" radio button under the "Has a collection order been made?" section
+        And I see "For example, 31/01/2023" hint text above the "Date of collection order" date picker
+        And I enter "20/05/2024" into the "Date of collection order" date field
+        When I select the "Pay in full" radio button
+        And I see "For example, 31/01/2023" hint text above the "Enter pay by date" date picker
+        And I enter "12/09/2023" into the "Enter pay by date" date field
+
+        Examples:
+            | dateOfBirth |
+            | 20/03/2000  |
+
+    Scenario Outline: AC-1a, AC-3 positive: the 'Has a collection order been made?' field should be presented only, if the defendant is an Adult
+        And I enter "20/03/2000" into the Date of birth field
+        And I click the "Return to account details" button
+        Then I see the status of "Personal details" is "Provided"
+        And I click on the "Payment terms" link
+        Then I see "Payment terms" on the page header
+
+        And I select the "No" radio button under the "Has a collection order been made?" section
+        And I select the "Make collection order today" checkbox
+        When I select the "Pay in full" radio button
+        And I see "For example, 31/01/2023" hint text above the "Enter pay by date" date picker
+        And I enter "12/09/2023" into the "Enter pay by date" date field
+
+    Scenario Outline: AC4 -negative: If the user has not selected a value for the 'Has a collection order been made?' field and clicks either the 'Return to account details' or 'Add account comments and notes' buttons
```

Comment:
not an issue with the test at all, and I know this isnt ideal but due to the time constraints we have in the pipelines, would you be able to combine these into a single scenario, I believe it should be fairly straightforward.

Reply:
I haven't created these 2 stpes. re-using from other feature files

Reply:
I kept these 2 steps in one scenarios and on the rest of the scenarios we don't necessarily need so removed

Reply:
In one step we are verifying the text presented on the screen, On the other step entering the value on to a specific field